fiftyone/utils/pe.py

```
@@ -0,0 +1,151 @@
+"""FiftyOne Integration with Perception Encoder by Meta.
+"""
+from typing import Optional
```

Pylint (4.0.3)

[warning] Unused Optional imported from typing

(W0611)